tests.py

```diff
@@ -383,9 +55,109 @@ def test(self):
         cls.cnt += 1
 
 
+UNSUPPORTED_OPERATIONS = (
+    "?:", "substr", "filter", "map", "reduce", "all", "none", "some")
```

Comment:
"?:". should not be in this list. You should not be removing the support for it. It should map to the same function as "if" does.